### ISSUE TITLE
[anchor-has-content] An error occurs even if the content is contained

### ISSUE THREAD

**Issue by yamanoku** [vue-a11y/eslint-plugin-vuejs-accessibility#68]
## Details
<img width="969" alt="Screen Shot 2020-09-21 at 19 52 45" src="https://user-images.githubusercontent.com/1996642/93759514-46439a80-fc45-11ea-91f5-4677bb86d539.png">

The Lint rule of `anchor-has-content` is triggered even if there is an `<img>` element (an `alt` attribute is also present).

## Version
```
"eslint-plugin-vuejs-accessibility": "0.5.1"
```

**Comment by yamanoku** [comment]
#66 similar issue?

**Comment by kddeisz** [comment]
Weird. This is a dup of https://github.com/vue-a11y/eslint-plugin-vuejs-accessibility/issues/48. I thought this had been fixed. I'm currently unable to reproduce this. It should be released in v0.5.0. You're 100% sure you're on the latest version?

**Comment by yamanoku** [comment]
I checked and it is the latest version.
https://github.com/yamanoku/yamanoku.github.io/blob/add-buymeacoffee-widget/package.json

Node.js is using the latest stable version, is there any other possible cause? 

The eslint configuration file is:
https://github.com/yamanoku/yamanoku.github.io/blob/add-buymeacoffee-widget/.eslintrc.js

**Comment by yamanoku** [comment]
@kddeisz 
Continued: I tried running lint, but there was no problem. It seems to be a display problem of VSCode 🤔 
Thank you for letting me know that you have already addressed this issue. I'll close the Issue here.

**Comment by vhoyer** [comment]
> @kddeisz
> Continued: I tried running lint, but there was no problem. It seems to be a display problem of VSCode 
> Thank you for letting me know that you have already addressed this issue. I'll close the Issue here.

It could be a problem of caching :D (it has happened to me before)
